config/config.go

```diff
@@ -29,6 +30,7 @@ func Init() {
 	config.ServerAddr = ":8000"
 	config.OpenApiSpecPath = "./spec/openapi.json"
 	config.Test = false
+	config.ProductName = os.Getenv("product_name") // Placeholder for the product name
```

Comment:
Correct me if I am wrong, but the template should be filled on the front end right?